src/cljs/commiteth/manage_payouts.cljs

```diff
@@ -304,9 +304,9 @@
 (defn count-pill [n]
   [:span.v-top.ml3.ph3.pv1.bg-black-05.gray.br3.f7 n])
 
-(defn salute [name]
```

Comment:
@martinklepsch just wanted to verify that this wasn't intended for defaulting or something? and can be safely removed?

Reply:
Oh whoops, seems I messed up some merge conflict. It should take the users name from the DB (instead of calling everybody `Andy` 😅  — will clean that up.)

Reply:
lol, cool. I think it does end up doing that and in this case "andy" is discarded. just wanted to verify. I can leave the removal in this current PR if that's easier and then you won't have to do anything :)